### ISSUE TITLE
Support streaming rtmp to gramrphone.live/

### ISSUE THREAD

**Issue by grahamtallen** [grahamtallen/GramrphoneWeb#21]

**Comment by grahamtallen** [comment]
https://github.com/arut/nginx-rtmp-module/issues/1280

**Comment by weepy** [comment]
i think this is not very important right for the moment as there's no user story that benefits ?

**Comment by grahamtallen** [comment]
Yes I agree that it’s not high priority right now. what’s best practice with depriotizing issues on github? I would imagine this would get frustrating eventually, if we don’t get this fixed and we move to something that has elastic ip’s, so I don’t necessarily want to close this ticket

**Comment by weepy** [comment]
agree. we can add priority levels as labels. if u make me an admin i can help too
also could make milestones to organise
